qancode/tasks.py

```diff
@@ -555,7 +556,7 @@ def make_experiment_pages_look_the_same(self):
     def get_data(self):
         self._try_load_item_type()
         self._try_perform_click_path()
-        if 'experiment' in self.driver.current_url:
+        if 'experiment' in self.driver.current_url.lower():
```

Comment:
What happens if `Experiment` in the URL and not `experiment`?

Reply:
Before introducing the lower(), QANCODE wouldn't click the file table on the experiment page if  the url had `Experiment` instead. I made this change because we added some item types for search queries followed by clicking an item (see the other comment) and I was running into the issue where it kept taking screenshots of the file graph.

Reply:
Sorry I read this wrong. Make sense. Let me know when you want to merge.